sos/report/plugins/aap_eda.py

```diff
@@ -0,0 +1,54 @@
+# Copyright (c) 2023 Rudnei Bertol Jr <rudnei@redhat.com>
+
+# This file is part of the sos project: https://github.com/sosreport/sos
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# version 2 of the GNU General Public License.
+#
+# See the LICENSE file in the source distribution for further information.
+
+from sos.report.plugins import Plugin, RedHatPlugin
+
+
+class AAPEDAControllerPlugin(Plugin, RedHatPlugin):
+    short_desc = 'AAP EDA Controller plugin'
+    plugin_name = 'aap_eda'
+    profiles = ('sysmgmt', 'ansible')
+    packages = ('automation-eda-controller',
+                'automation-eda-controller-server')
+
+    def setup(self):
+        self.add_copy_spec([
+            "/etc/ansible-automation-platform/",
+            "/var/log/ansible-automation-platform/eda/worker.log*",
+            "/var/log/ansible-automation-platform/eda/scheduler.log*",
+            "/var/log/ansible-automation-platform/eda/gunicorn.log*",
+            "/var/log/ansible-automation-platform/eda/activation.log*",
+        ])
+
+        self.add_forbidden_path([
+            "/etc/ansible-automation-platform/eda/SECRET_KEY",
+            "/etc/ansible-automation-platform/eda/server.cert",
+            "/etc/ansible-automation-platform/eda/server.key",
+        ])
+
+        self.add_cmd_output([
+            "aap-eda-manage --version",  # eda version
+            "ls -alhR /etc/ansible-automation-platform/",
+            "ls -alhR /var/log/ansible-automation-platform/",
+        ])
+
+        self.add_cmd_output("su - eda -c 'podman ps -a'",
+                            suggest_filename="eda_podman_ps_-a")
+
+        self.add_cmd_output("su - eda -c 'export'",
+                            suggest_filename="eda_export")
```

Comment:
We're seeing requests for non-root collections come up over time. I think we need to build out an actual mechanism for this in `add_cmd_output()`, rather than running a bunch of `su` commands in plugins.

Reply:
Indeed; originally I even though we already have an option of `a_c_o` like `su_user` when reviewing this plugin :) . Nowadays I see `saphana`, `sapnw` and `vdsm` plugins calling commands under `su`.

I guess we can/shall propose such option independently on this PR.